src/main/java/com/commercetools/emailprocessor/Main.java

```diff
@@ -0,0 +1,68 @@
+package com.commercetools.emailprocessor;
+
+
+import com.commercetools.emailprocessor.email.EmailProcessor;
+import com.commercetools.emailprocessor.model.ProjectConfiguration;
+import com.commercetools.emailprocessor.model.Statistics;
+import com.commercetools.emailprocessor.model.TenantConfiguration;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.sphere.sdk.json.SphereJsonUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+
+
+public class Main {
+    /**
+     * This jobs try to resend emails of a given List of tenant using the EmailProcessor
+     * The configuration of this job, can be pass by  a File or enviroment variable.
+     *
+     * @param args all args
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+    public static String CTP_PROJECT_CONFIG = "git";
+
+    public static void main(final String[] args) {
```

Comment:
You could split this long method into shorter ones.

Reply:
7fead24